File: Problems/12900/Problem.js
function solution(n) {    
    let count = [1, 1, 2, 3, 5, 8];
    
    for (let i=2; i<=n; i++) {
        count[i] = (count[i-1] + count[i-2]) % 1000000007;
    }
    
    return count[n];
}

console.log(solution(7));

File: Problems/12907/Problem.py
def solution(n, money):
    answer = 0
    changes = [0] * 100001
    changes[0] = 1
    for m in money:
        for count in range(1, n+1):
            if count - m >= 0:
                changes[count] += changes[count - m] % 1000000007
    
    #for k in range(0, n+1):
        #print(changes[k])
    
    answer = changes[n]
    return answer

print(solution(5, [1,2,5]))
